core/templates/components/filter-fields/filtered-choices-field/filtered-choices-field.component.ts

```diff
@@ -37,13 +37,16 @@ export class FilteredChoicesFieldComponent {
 
   filteredChoices!: string[];
 
+  constructor(private cd: ChangeDetectorRef) {}
```

Comment:
Can you call it `changeDetectorRef` instead of `cd`?

Reply:
Done